### DISCUSSION TITLE
2022 年终总结 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#14] [General]
# 2022 年终总结 | 高策

江湖小虾米

http://gaocegege.com/Blog/%E9%9A%8F%E7%AC%94/newyear2022

**Comment by zwpaper** [comment]
看到你年终总结才反应过来，当时给 envd 的贡献居然已经是 8 月份了，这一年都要过去了，果然疫情+养娃+大肚三重 buff 的叠加下，根本感知不到时光飞逝🤣

**Reply by gaocegege** [reply]
是啊，感觉在急速衰老

**Comment by grokking88** [comment]
真好。
